src/Raven.Client/Documents/Operations/AI/Agents/AiAgentPersistenceConfiguration.cs

```diff
@@ -19,28 +19,28 @@ public AiAgentPersistenceConfiguration()
         /// Initializes a new instance of the <see cref="AiAgentPersistenceConfiguration"/> class
         /// with the specified target collection and optional expiration.
         /// </summary>
-        /// <param name="collection">
-        /// The name of the database collection where chat session documents should be stored.
-        /// This is typically a collection like "Chats" or "Conversations".
+        /// <param name="conversationIdPrefix">
+        /// The prefix of the conversation ID.
+        /// This is typically a collection like "Chats/" or "Conversations/".
```

Comment:
remove 'collection' and lowercase the examples?

Reply:
done